src/Marksimple.php

```diff
@@ -11,6 +11,13 @@
 
 class Marksimple
 {
+    /**
+     * Error log level number
+     *
+     * @var integer
+     */
+    private $errorLogLevel = 200;
```

Comment:
Should we not use a integer from the psr-3 specification? like `error`. - https://www.php-fig.org/psr/psr-3/#5-psrlogloglevel